code/datums/helper_datums/command_alerts.dm

```diff
@@ -316,7 +316,7 @@
 	message = {"
 There's been a galaxy-wide electromagnetic pulse.  All of our systems are heavily damaged and many personnel are dead or dying. We are seeing increasing indications of the universe itself beginning to unravel.
 
-[station_name()], you are the only facility nearby a bluespace rift, which is near your research outpost.  You are hereby directed to enter the rift using all means necessary, quite possibly as the last humans alive.
+[station_name()], you are the only facility nearby a bluespace rift, which is near the [get_area_name(global_cascade_portal, 1)].  You are hereby directed to enter the rift using all means necessary, quite possibly as the last humans alive.
```

Comment:
```suggestion
[station_name()], you are the only facility nearby a bluespace rift, which is near the [get_area_name(global_cascade_portal, 1)] at your Research Outpost.  You are hereby directed to enter the rift using all means necessary, quite possibly as the last humans alive.
```

Reply:
Most players might not think of the Research Outpost first when they read things such as "Isolation Cell 2", "Auxiliary Storage" or "Break Room". It doesn't hurt to specify.

Reply:
Deff's portal doesn't spawn on the research outpost.

Reply:
neither does roids. this is literally why I removed that.